Modules/@babylonjs/react-native/EngineHook.ts

```diff
@@ -69,9 +69,12 @@ if (Platform.OS == "windows") {
     WebXRExperienceHelper.prototype.enterXRAsync = async function (...args: any[]): Promise<any> {
         // TODO: https://github.com/BabylonJS/BabylonNative/issues/577
         // Windows HMDs require different rendering behaviors than default xr rendering for mobile devices
-        await originalEnterXRAsync.apply(this, args);
+        const sessionManager = await originalEnterXRAsync.apply(this, args);
         this.scene.clearColor = Color3.Black().toColor4();
         this.scene.autoClear = true;
+        return new Promise<any>((resolve) => {
+            resolve(sessionManager);
```

Comment:
nit: I would put everything in the promise-- just feels a little strange to have a promise resolve immediately with work that happens outside of its function body thing but inside the function call returning the promise which represents the async work being done to create sessionManager